src/components/button/button.ts

```diff
@@ -1,11 +1,14 @@
 import {
-  Component,
-  ViewEncapsulation,
-  Input,
-  HostBinding,
-  HostListener,
-  ChangeDetectionStrategy,
+    Component,
+    ElementRef,
+    ViewEncapsulation,
+    Input,
+    HostBinding,
+    HostListener,
+    ChangeDetectionStrategy,
+    Renderer,
```

Comment:
As mentioned in the style conventions, you should not indent with a tab, better indent with two spaces.